Dockerfile.armhf

```diff
@@ -7,21 +7,7 @@ ARG CALIBRE_RELEASE
 LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DATE}"
 LABEL maintainer="aptalca"
 
-ENV \
-  CUSTOM_PORT="8080" \
-  GUIAUTOSTART="true" \
-  HOME="/config"
-
-RUN \
-  echo "**** install runtime packages ****" && \
-  pacman -Sy --noconfirm --needed \
-    calibre && \
-  dbus-uuidgen > /etc/machine-id && \
-  echo "**** cleanup ****" && \
-  rm -rf \
-    /tmp/* \
-    /var/cache/pacman/pkg/* \
-    /var/lib/pacman/sync/*
-
 # add local files
-COPY root/ /
+COPY root-armhf/ /
```

Comment:
Shouldn't it stay `root/`

Reply:
No, it's simpler to add a different root folder structure for armhf when we don't have an image for it because it avoids having to use conditionals that might not trigger for whatever reason. but it does need to stop the underlying rdesktop services which I forgot.